diff --git a/dist/index.mjs b/dist/index.mjs
index 92a80377692488c4ba8801ce33e7736ad7055e43..07443a47f422e952468d88beb517877b1ae6961e 100644
--- a/dist/index.mjs
+++ b/dist/index.mjs
@@ -504,7 +504,7 @@ function selectTelemetryAttributes({
     return {};
   }
   return Object.entries(attributes).reduce((attributes2, [key, value]) => {
-    if (value === void 0) {
+    if (value == null) {
       return attributes2;
     }
     if (typeof value === "object" && "input" in value && typeof value.input === "function") {
